docs/NON_DEVCONTAINER_SETUP.md

```diff
@@ -1,54 +1,80 @@
 [Back to *Chat with your data* README](../README.md)
 
-# Project setup
+# Non-DevContainer Setup
 
-If you are unable to run this accelerator as a DevContainer or in CodeSpaces, then you will need.
+If you are unable to run this accelerator using a DevContainer or in GitHub CodeSpaces, then you will need to install the following prerequisites on your local machine.
 
-- [Visual Studio Code](https://code.visualstudio.com/)
-    - Extensions
-        - [Azure Functions](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions)
-        - [Azure Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack)
-        - [Bicep](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-bicep)
-        - [Pylance](https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance)
-        - [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
-        - [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension) (optional: Teams extension only)
-- Install [Node.js](https://nodejs.org/en)
-  - Install the LTS version (Recommended for Most Users)
+- A code editor. We recommend [Visual Studio Code](https://code.visualstudio.com/), with the following extensions:
+  - [Azure Functions](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions)
+  - [Azure Tools](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack)
+  - [Bicep](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-bicep)
+  - [Pylance](https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance)
+  - [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
+  - [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension) **Optional**
+- [Python 3.11](https://www.python.org/downloads/release/python-3119/)
+- [Node.js LTS](https://nodejs.org/en)
+- [Azure Developer CLI](https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/install-azd)
+- [Azure Functions Core Tools](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local)
 
+## Setup
 
-### Running the sample using the Azure Developer CLI (azd)
+1. Configure your Python environment:
 
-The Azure Developer CLI (`azd`) is a developer-centric command-line interface (CLI) tool for creating Azure applications.
+    ```bash
+    pip install --upgrade pip
+
+    pip install poetry
+
+    # https://pypi.org/project/poetry-plugin-export/
```

Comment:
Is this required?

Reply:
Poetry needs to be installed, yes. We don't call it out as a pre-req. Why do you think it is redundant?

Reply:
Ah sorry, I meant the comment!
```suggestion
```